### DISCUSSION TITLE
Quorum queue unavailable (still in pre_vote state) after rolling rollback v3.11.13 -> v3.10.20

### DISCUSSION THREAD

**Discussion by briceviard** [rabbitmq/rabbitmq-server#7934] [Other] [answered]
### Describe the bug

We are testing rolling upgrade and rolling rollback with rabbitmq version 3.10.20 and 3.11.13.
No problem with the rolling upgrade to v3.11.13 but after the rolling downgrade to v3.10.20 all quorum queues are unavailable, they still in a pre_vote state and java consumer can't recover consumer...

[javaconsumer.log](https://github.com/rabbitmq/rabbitmq-server/files/11270539/javaconsumer.log)

[rabbitmq-node.log](https://github.com/rabbitmq/rabbitmq-server/files/11270540/rabbitmq-node.log)

[queue-status.log](https://github.com/rabbitmq/rabbitmq-server/files/11270550/queue-status.log)

### Reproduction steps

1. create a quorum queue on rabbitmq cluster v3.10.20
2. start a java consumer on the quorum queue
3. rolling upgrade rabbitmq cluster from 3.10.20 to 3.11.13 
4. rolling rollback rabbitmq cluster from 3.11.12 to 3.10.20
5. The java consumer has been disconnected and can't recover de connection
6. The quorum queue stuck in the pre_vote state
...

### Expected behavior

To do a rolling rollback without downtime with all quorum queue available

**Comment by mkuratczyk** [comment, marked as answer]
https://www.rabbitmq.com/upgrade.html
`RabbitMQ does not support downgrades; it's strongly advised to back node's data directory up before upgrading.`

We never promised something like that would work...
